marklogic-data-hub-central/ui-custom/src/components/confirmation-modal/confirmation-modal.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import {Modal} from "react-bootstrap";
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {faExclamationTriangle} from "@fortawesome/free-solid-svg-icons";
+import "./confirmation-modal.scss";
+
+type Props = {
+  isVisible: boolean;
+  toggleModal: (isVisible: boolean) => void;
+  confirmAction: () => void;
+  bodyContent: string;
+  headerContent: string;
+};
+
+const ConfirmationModal: React.FC<Props> = (props) => {
+
+  const closeModal = () => {
+    props.toggleModal(false);
+  };
+
+  const handleYesClicked = () => {
+    props.confirmAction();
+    props.toggleModal(false);
+  }
+
+  return (
+    <Modal
+      show={props.isVisible}
+      data-testid="resetConfirmationModal"
+    >
+      {props.headerContent.length !== 0 &&<Modal.Header>props.headerContent</Modal.Header>}
+      <Modal.Body>
+      <div className="modalBody">
+        <FontAwesomeIcon icon={faExclamationTriangle} className="alertIcon"/>
+        <span className="bodyContent">{props.bodyContent}</span>
+      </div>
+    </Modal.Body>
+      <Modal.Footer>
+        <button className="submit" onClick={closeModal} data-testid="noButton">No</button>
```

Comment:
Per Julia's design spec (see the ticket screenshot), please give the secondary button (in this case the "No" cancel button) a dark gray label and white background. You can give it a "cancel" class.